### DISCUSSION TITLE
'parseMutationArgs' is not exported from '@tanstack/react-query'

### DISCUSSION THREAD

**Discussion by hoanmq** [TanStack/query#6251] [Q&A] [answered]
I faced with bellow issue after upgrade from v4 to v5

```   
"@tanstack/react-query": "^5.0.5",
"@tanstack/react-query-devtools": "^5.4.2",
```

Attempted import error: 'parseMutationArgs' is not exported from '@tanstack/react-query' (imported as 'parseMutationArgs').
ERROR in ./node_modules/react-query-kit/build/lib/createMutation.mjs 4:25-42
export 'parseMutationArgs' (imported as 'parseMutationArgs') was not found in '@tanstack/react-query' (possible exports: CancelledError, HydrationBoundary, InfiniteQueryObserver, IsRestoringProvider, MutationCache, MutationObserver, QueriesObserver, Query, QueryCache, QueryClient, QueryClientContext, QueryClientProvider, QueryErrorResetBoundary, QueryObserver, defaultShouldDehydrateMutation, defaultShouldDehydrateQuery, dehydrate, focusManager, hashKey, hydrate, infiniteQueryOptions, isCancelledError, isServer, keepPreviousData, matchQuery, notifyManager, onlineManager, queryOptions, replaceEqualDeep, useInfiniteQuery, useIsFetching, useIsMutating, useIsRestoring, useMutation, useMutationState, useQueries, useQuery, useQueryClient, useQueryErrorResetBoundary, useSuspenseInfiniteQuery, useSuspenseQueries, useSuspenseQuery)

**Comment by DamianOsipiuk** [comment, marked as answer]
This is an issue with `react-query-kit` that is probably not updated yet to support v5.
There is nothing we can do here.
